src/diffusers/pipelines/animatediff/pipeline_animatediff.py

```diff
@@ -326,6 +326,8 @@ def encode_prompt(
 
     # Copied from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion.StableDiffusionPipeline.encode_image
     def encode_image(self, image, device, num_images_per_prompt, output_hidden_states=None):
```

Comment:
```suggestion
    def encode_image(
```
Sorry we'll have to manually fix this for all occurances under #Copied from I'm afraid

Reply:
Ah I see. I'll fix it today